manifests/charts/cloudcore/templates/configmap_cloudcore.yaml

```diff
@@ -6,6 +6,12 @@ metadata:
   labels: {{- toYaml . | nindent 4 }}
   {{- end }}
 data:
+  {{- if (ne .Values.cloudCore.configFile "") }}
+  {{- $files := .Files }}
+  {{- $config := .Values.cloudCore.configFile -}}
+  cloudcore.yaml: |
+      {{ $files.Get "$config" }}
```

Comment:
@zhu733756  hi, do you know how to access a file outside `charts` directory, e.g. /etc/kubeedge/config/cloudcore.yaml. I found files.Get can only get the content of files in charts directory.

Reply:
I see a function like here in https://helm.sh/docs/chart_template_guide/accessing_files/  .
The page says that "Then we print each file name ({{ . }}: |-) followed by the contents of the file {{ $files.Get . }}."
Can you try it?

Reply:
we cannot access the files outside chart directory , by asking in helm slack, the answer is below
`
That doesnt work by design. Imagine you give this chart to your colleage. If they dont have the file in the same place, the chart is broken. It is probabably also a security risk. Imagine some third party chart goes out and fetches your /etc/password file.
`

Reply:
Got it. I think they are correct.